pkg/pool-smart/test/LiquidityBootstrappingPool.test.ts

```diff
@@ -0,0 +1,753 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { BigNumber } from 'ethers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { MinimalSwapInfoPool, TwoTokenPool } from '@balancer-labs/v2-helpers/src/models/vault/pools';
+import { BigNumberish, bn, fp, pct } from '@balancer-labs/v2-helpers/src/numbers';
+import { ZERO_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+
+import TokenList from '@balancer-labs/v2-helpers/src/models/tokens/TokenList';
+import LiquidityBootstrappingPool from '@balancer-labs/v2-helpers/src/models/pools/smart/LiquidityBootstrappingPool';
+import { RawLiquidityBootstrappingPoolDeployment } from '@balancer-labs/v2-helpers/src/models/pools/smart/types';
+
+describe('LiquidityBootstrappingPool', function () {
+  let allTokens: TokenList;
+  let trader: SignerWithAddress, recipient: SignerWithAddress;
+  let other: SignerWithAddress, lp: SignerWithAddress, owner: SignerWithAddress;
+
+  const POOL_SWAP_FEE_PERCENTAGE = fp(0.01);
+  const WEIGHTS = [fp(30), fp(70), fp(5), fp(5)];
```

Comment:
```suggestion
  const WEIGHTS = [fp(30), fp(65), fp(10), fp(5)];
```

Let's make all weights different so that we can be sure they are being stored and read correctly.